caddyhttp/staticfiles/fileserver.go

```diff
@@ -168,3 +200,13 @@ var IndexPages = []string{
 	"default.htm",
 	"default.txt",
 }
+
+var staticEncoding = map[string]string{
+	"gzip": ".gz",
+	"br":   ".br",
+}
+
+var staticEncodingPriority = []string{
+	"br",
+	"gzip",
+}
```

Comment:
Can you comment on what these variables do? godoc style (I know they're unexported, but it's not obvious what they're used for, since this isn't the compression module.)